modules/acquia_cms_common/acquia_cms_common.install

```diff
@@ -15,8 +15,9 @@ use Drupal\workbench_email\Entity\Template;
  * Implements hook_install().
  */
 function acquia_cms_common_install($is_syncing) {
-  $module_handler = \Drupal::moduleHandler();
   if (!$is_syncing) {
+    $module_handler = \Drupal::moduleHandler();
+    $moduleInstaller = \Drupal::service('module_installer');
```

Comment:
@rajeshreeputra Use snake_case for .install .module files 

cc: @vishalkhode1 @chandan-singh7929